cdap-hbase-compat-base/src/main/java/co/cask/cdap/replication/TableUpdater.java

```diff
@@ -111,10 +118,26 @@ public void run() {
     updateTimer.scheduleAtFixedRate(updateTask, delay, period);
   }
 
+  private void markMapDirty(HashMap<String, TimeValue> updateMap) {
+    for (Map.Entry<String, TimeValue> entry : updateMap.entrySet()) {
+      entry.getValue().stale = true;
+    }
+  }
+
+  private HashMap<String, Long> getNewEntries(HashMap<String, TimeValue> updateMap) {
+    HashMap<String, Long> cleanMap = new HashMap<>();
```

Comment:
Map<String, Long> cleanMap. Usually you use the more generic interface instead of the actual implementation